scripts/subgraph/getSaleHistory.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * An example of how to query the Foundation sale history from our hosted subgraph.
+ * Run with `yarn run-script scripts/subgraph/getSaleHistory.ts`
+ *
+ * See `scripts/rpc/getSaleHistory.ts` for an alternative approach to querying the Foundation sale history.
+ */
+
+import gql from "graphql-tag";
+import { GraphQLClient } from "graphql-request";
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const subgraphEndpoints = require("@f8n/fnd-protocol/subgraphEndpoints.js");
+
+// The example NFT to query for, this should be updated with the NFT of interest
+const collection = "0x3B3ee1931Dc30C1957379FAc9aba94D1C48a5405";
```

Comment:
just an idea, how about making these use ENV variables (process.env...) if provided else default to this hardcoded address.

this would help ppl just wanting to run the script rather than copy pasting /understanding the blob

Reply:
It's an arbitrary contract we're using for the scripts, I picked it cause it was the first I found with > 1 sale. I think an ENV var just complicates the example a bit and doesn't really make sense to use here. The idea was you could just replace it with the contract/tokenId of interest to see it work